Clarify requirements for running examples (#53143)
vercel/next.js: contributing/examples/run-example-apps.md

pnpm next-with-deps ./examples/basic-css/
```

> > Before you can run the example, you need to install all the dependencies with `pnpm i` and build the `next.js` by running `yarn build` or `pnpm run build` in the root directory.

Reviewer: ```suggestion
> Before running any of the examples, you need to install dependencies via `pnpm install` and build the repository packages by running `pnpm build` in the root directory.
```
